src/arrays.js

```diff
@@ -1,61 +1,157 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const maxArrayindex = (array.length - 1);
+  // console.log(maxArrayindex);
+  const indexGiven = index;
+  // console.log(indexGiven);
+
+  if (indexGiven > maxArrayindex) {
+    const newIndex = indexGiven - 4;
+    // console.log('here');
+    return array[newIndex];
+  }
+  return array[indexGiven];
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  const csvArray = array.toString();
```

Comment:
you don't need to assign the result to a new variable before returning, you can simply return the result

```
return array.toString();
```

Reply:
you can also achieve this using `.join` method on Array prototype

```
return array.join(', ');
```